src/Symfony/Upgrade/Fixer/FormGetnameToGetblockprefixFixer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Symfony\Upgrade\Fixer;
+
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+class FormGetnameToGetblockprefixFixer extends FormTypeFixer
+{
+
+    /**
+     * Fixes a file.
+     *
+     * @param \SplFileInfo $file A \SplFileInfo instance
+     * @param string       $content The file content
+     *
+     * @return string The fixed file content
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        if ($this->isFormType($tokens) && null !== $this->matchGetNameMethod($tokens)) {
```

Comment:
I copy/pasted most of the logic from here https://github.com/umpirsky/Symfony-Upgrade-Fixer/blob/master/src/Symfony/Upgrade/Fixer/FormConfigureOptionsFixer.php .

I removed the check in the method.